plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui.test/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/internal/ProjectSelectorSelectionChangedListenerTest.java

```diff
@@ -42,7 +42,8 @@
 public class ProjectSelectorSelectionChangedListenerTest {
 
   private static final String EXPECTED_LINK =
-      "https://console.cloud.google.com/appengine/create?lang=java&project=projectId";
+      "https://console.cloud.google.com/appengine/create?lang=java&project=projectId"
+      + "&authuser=user@example.com";
```

Comment:
I did some investigation, and my conclusion is that, although `@` may work in most systems and browsers, but it should be encoded.